documentation/asciidoc/computers/configuration/host-wireless-network.adoc

```diff
@@ -0,0 +1,155 @@
+== Host a wireless network on your Raspberry Pi
+
+Your Raspberry Pi can host its own wireless network using a wireless module.
+If you connect your Raspberry Pi to the internet via the Ethernet port (or a second wireless module), other devices connected to the wireless network can access the internet through your Raspberry Pi.
+
+Consider a wired network that uses the `10.x.x.x` IP block. You can connect your Raspberry Pi to that network and serve wireless clients on a separate network that uses another IP block, such as `192.168.x.x`. In the diagram below, note that the laptop exists in an IP block separate from the router and wired clients:
+
+----
```

Comment:
Can you ping @jackbenwillis about these diagrams? I think we'd want proper artwork in here. I know the original tutorials had ASCII art, but we're trying to do a bit better about things these days!